apps/example/package.json

```diff
@@ -48,6 +48,7 @@
     "starknet": "6.6.6",
     "tailwind-merge": "^2.1.0",
     "tailwindcss-animate": "^1.0.7",
+    "viem": "^2.12.1",
     "web3": "^4.4.0",
```

Comment:
web3 should be remove now

Reply:
Web3 is still used in apps/example, we might fully remove it via another PRs.